apis/python/tools/cartographer

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env python
```

Comment:
this file is very opinionated about the higher level schema (defined above the SOMA container API), and not really a SOMA tool.  Suggest that it be moved to the examples directory, and clearly documented for what it is.  There really is no such thing as a generic SOMA-level "atlas", as any atlas makes choices about how to integrate data.

if you would like a general purpose SOMA tool, perhaps a completely unopinionated SOCA creator that just add the entirety of a SOMA or H5AD to the SOCO?

Reply:
> Suggest that it be moved to the examples directory, and clearly documented for what it is. 

awesome @bkmartinjr -- this is exactly the kind of feedback i was looking for.

> perhaps a completely unopinionated SOCA creator that just add the entirety of a SOMA or H5AD to the SOCO?

this will be a straightforward combine of the existing `tools/ingestor` and `tools/populate-soco` -- long-overdue for a merge :)